README.md

```diff
@@ -17,12 +17,17 @@ $ docker pull namely/protoc-all:1.9
 After that, travel to the directory that contains your `.proto` definition
 files.
 
-So if you have a directory: `/Users/me/project/protobufs/` that has:
+So if you have a directory: `~/my_project/protobufs/` that has:
 `myproto.proto`, you'd want to do this:
 
 ```sh
-cd ~/my_project/protobufs
-docker run -v `pwd`:/defs namely/protoc-all:1.9 -f myproto.proto -l ruby #or go, csharp, etc
+$ cd ~/my_project/protobufs
+$ docker run -v `pwd`:/defs namely/protoc-all:1.9 -f myproto.proto -l ruby #or go, csharp, etc
```

Comment:
I would have to double-check, but believe this won't work on windows without first mounting `/mnt/c` to `/c`. https://nickjanetakis.com/blog/setting-up-docker-for-windows-and-wsl-to-work-flawlessly#bind-custom-mount-points-to-fix-docker-for-windows-and-wsl-differences

Reply:
updated.